pyproject.toml

```diff
@@ -33,4 +33,4 @@ mark-parentheses = false
 
 [tool.ruff.lint.pep8-naming]
 # if overriding a PyQt method, please add it here!
-extend-ignore-names = ["allKeys", "createEditor", "setEditorData", "textFromValue"]
+extend-ignore-names = ["allKeys", "createEditor", "setEditorData", "textFromValue", "mergeWith"]
```

Comment:
```suggestion
# names should be in alphabetical order for readability
extend-ignore-names = ["allKeys", "createEditor", "mergeWith", "setEditorData", "textFromValue", ]
```
sorry to be pedantic: i was trying to keep these in alphabetical order so that it's easy to look through if needed!